src/ui/components/LedgerViewer.js

```diff
@@ -59,8 +65,11 @@ export const LedgerViewer = ({
   const {ledger} = useLedger();
   const classes = useStyles();
   const [allocation, setAllocation] = useState<Allocation | null>(null);
-  const [page, setPage] = useState(0);
-  const [rowsPerPage, setRowsPerPage] = React.useState(25);
+  const [DATE_SORT] = useState({
+    name: "Date",
+    fn: (e: LedgerEvent) => e.ledgerTimestamp,
+  });
+  const [PAGINATION_OPTIONS] = useState([25, 50, 100]);
```

Comment:
These are both using useState, but the state is never being changed here (no second `setPaginationOptions` or `setDateSort` options assigned in the return value). If the state is never updated, you can just define the value outside of the LedgerViewer component as a constant.

Reply:
aha, accepted